### PR TITLE
8.1.x Sync

### PR DESCRIPTION
**Pull request by datamel**

### CODE REVIEW

File: cylc/flow/__init__.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def environ_init():
 
 environ_init()
 
-__version__ = '8.1.1.dev'
+__version__ = '8.2.0.dev'
```
Comment (wxtim) [inline cylc/flow/__init__.py:49]: Surely not?

Comment (datamel) [reply cylc/flow/__init__.py:49]:
```suggestion
__version__ = '8.1.1.dev'
```

**Comment by oliver-sanders** [comment]
Because this PR is raised from the `8.1.x` branch, suggestions applied here are applied to the `8.1.x` branch.

To get around this, if there are conflicts to be resolved with a sync PR we have to create an intermediate branch like this:

```console
$ git checkout -b sync upstream/master
$ git merge upstream/8.1.x
$ git push origin sync
```

**Comment by MetRonnie** [comment]
The problem here was the PR is master into 8.1.x - the wrong way round! Also it seems the branch protection rules are blocking force pushes but allowing normal pushes for some reason! Looking into it

**Comment by oliver-sanders** [comment]
Cheers.

We could do with automating this at some point.

**Pull request closed without merging**